kpi/maintenance_tasks.py

```diff
@@ -26,3 +26,16 @@ def remove_old_asset_snapshots():
         ).delete()
         if not count:
             break
+
+
+def remove_old_import_tasks():
+    delete_queryset = ImportTask.objects.filter(
+        date_created__lt=timezone.now() - timedelta(days=90),
```

Comment:
@noliveleger in the original PR you wrote "(90 by default)" is there any reason to make this configurable? I like hard coding it, if not. Would a private server ever say want to make it longer?

Reply:
It's part of an old discussion of the back-end team. IIRC, yes some private servers wanted a different value. So please keep it configurable.

Reply:
Added constance var